tests/integration/test_notes.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from conftest import is_valid
+from models.notes import NotesModel
+
+
+@pytest.mark.usefixtures("client_class", "empty_test_db")
+class TestCreate:
+    def setup(self):
+        self.endpoint = "/api/tickets"
+        self.newNote = {"text": "We don't need no water"}
```

Comment:
minor style: The backend uses snake_case. Please use that style everywhere.

Reply:
Fixed, thanks!